src/welcome/admin.js

```diff
@@ -401,6 +402,20 @@ const EditorSettings = () => {
 			} }
 			help={ __( 'Disable this if you encounter layout or spacing issues when using images inside carousel-type blocks because of image lazy loading.', i18n ) }
 		/>
+		<AdminToggleSetting
+			label={ __( 'Inherit Theme Styles', i18n ) }
```

Comment:
Let's make this specific to buttons for now
```suggestion
			label={ __( 'Inherit Theme Button Styles', i18n ) }
```